pkg/redfish/redfish-support.go

```diff
@@ -45,8 +47,34 @@ func (rs ResourceIDSlice) Len() int {
 	return len(rs)
 }
 
+var alphanum = regexp.MustCompile("([a-zA-Z]+)([0-9]+)$")
+
 // For sort interface, comparison operation
 func (rs ResourceIDSlice) Less(i, j int) bool {
```

Comment:
Id like to see unit testing on this function.   Happy path; sad path (junk data).

Reply:
Added two tests for sorting of Resource IDs.  Sort should always work, just a matter of which way they are sorted.

Reply:
MustComplie will panic if it doesnt compile... which means what?  some unit of execution will die. Is it the whole service or just some thread. https://pkg.go.dev/regexp#MustCompile

Reply:
At startup anything that uses this package will cause the panic to happen (for this particular case), if the regular expression is invalid. FWIW RTS makes use of the MustCompile a lot